src/overlays/fbdemos/ovl_fbdemo_wipe4/z_fbdemo_wipe4.c

```diff
@@ -1,11 +1,22 @@
-/*
- * File: z_fbdemo_wipe4.c
+/**
+ * @file z_fbdemo_wipe4.c
  * Overlay: ovl_fbdemo_wipe4
  * Description:
+ *
+ * @note The bug in TransitionWipe4_Draw() makes this transition do nothing apart from increase its timer until it is
+ * finished.
  */
 
 #include "global.h"
 #include "z_fbdemo_wipe4.h"
+#include "sys_cfb.h"
+
+#define THIS ((TransitionWipe4*)thisx)
+
+// TODO: determine how generic these macros are. Requires rest of the fbdemos decompiled.
+#define TW4_SET_PARAMS (1 << 7)
```

Comment:
Instead of defining a new macro, should just use `TRANS_TYPE_SET_PARAMS`

Reply:
Also since I noticed it, could you add a parens around `type` in the `FBDEMO_CIRCLE_GET_MASK_TYPE` macro in `z64transition.h`.